doc/generic/pgf/pgfmanual-en-guidelines.tex

```diff
@@ -645,7 +645,7 @@ \subsection{Attention and Distraction}
     \item Background patterns filling an area using  diagonal lines or
         horizontal and vertical lines or just dots are almost always
         distracting and, usually, serve no real purpose.
-    \item Background images and shadings distract and only seldomly add
+    \item Background images and shadings distract and only seldom add
```

Comment:
doubtful, or seems unnecessary. According to Merriam-Webster [\[1\]][1],

> _Seldomly_ is a perfectly logical adverb form, but it has been labeled “archaic” or “rare” in our dictionaries for over a century (our 19th-century editions didn’t even enter it). Seldom is itself an adverb (“We seldom go out on weeknights”), making the _-ly_ form redundant and unnecessary. It has been out of standard use for so long that it is rarely encountered in publications with national readership, but it still sometimes pops up:
> 
> > And there are the requisite pool and tennis court, which are seldomly used.
> > —The New York Times, 6 July 1997
> > 
> > Apple is wise enough to introduce new colours so seldomly that fans go into meltdown each time a new hue is inducted into the fold.
> > —The Telegraph, 22 September 2015

[1]: https://www.merriam-webster.com/words-at-play/6-common-hypercorrections-and-how-to-avoid-them